app/src/main/java/com/codesoom/assignment/App.java

```diff
@@ -1,10 +1,34 @@
 package com.codesoom.assignment;
 
+import com.codesoom.assignment.handlers.DemoHttpHandler;
+import com.codesoom.assignment.handlers.TaskHttpHandler;
+import com.sun.net.httpserver.HttpHandler;
+import com.sun.net.httpserver.HttpServer;
+
+import java.io.IOException;
+import java.net.InetSocketAddress;
+
 public class App {
-    public String getGreeting() {
-        return "Hello World!";
-    }
+
+    private static final int PORT = 8000;
+    private static final int BACK_LOG = 0;  // TCP 연결의 최대 값. 0 혹은 0보다 작을 경우 system default 값으로 설정된다.
 
     public static void main(String[] args) {
+        try {
+
+            InetSocketAddress address = new InetSocketAddress(PORT);
+            HttpServer httpServer = HttpServer.create(address, BACK_LOG);
+
+            HttpHandler handler = new DemoHttpHandler();
+            TaskHttpHandler taskHttpHandler = new TaskHttpHandler();
```

Comment:
작은 범위이긴 하지만 `final`을 붙이는 습관을 들여 봅시다.
```suggestion
            final InetSocketAddress address = new InetSocketAddress(PORT);
            final HttpServer httpServer = HttpServer.create(address, BACK_LOG);

            final HttpHandler handler = new DemoHttpHandler();
            final TaskHttpHandler taskHttpHandler = new TaskHttpHandler();
```

Reply:
✅ `final`을 붙였습니다. 명확한 이유에 대해서는 설명할 수 있을 정도로 조금 더 공부해봐야겠네요!